style.css

```diff
@@ -70,3 +71,250 @@ body {
   list-style: none;
   padding: 0;
 }
+
+.scroll {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.box-contain {
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+  background-color: #fff;
+  list-style: none;
+  margin-bottom: 50px;
```

Comment:
Please ensure that for each project the content is aligned with the flexbox according to project requirement